src/opendr/perception/object_tracking_2d/deep_sort/object_tracking_2d_deep_sort_learner.py

```diff
@@ -418,7 +418,8 @@ def _prepare_datasets(
         def map_dataset(dataset):
 
             def image_to_pil(image: Image):
```

Comment:
Eventually, we should probably make this generic. E.g., I have a similar function in my code.

Reply:
Perhaps we could add an "pil()" method to Image and also try to perform a cleanup in the code to reduce duplicate conversions that I think happen in some learners.